cvat-ui/src/components/annotation-page/canvas/grid-layout/canvas-layout.tsx

```diff
@@ -72,7 +72,10 @@ const fitLayout = (type: DimensionType, layoutConfig: ItemLayout[]): ItemLayout[
     const relatedViews = layoutConfig
         .filter((item: ItemLayout) => item.viewType === ViewType.RELATED_IMAGE);
     const relatedViewsCols = relatedViews.length > 6 ? 2 : 1;
-    const height = Math.floor(config.CANVAS_WORKSPACE_ROWS / (relatedViews.length / relatedViewsCols));
+    let height = Math.floor(config.CANVAS_WORKSPACE_ROWS / (relatedViews.length / relatedViewsCols));
+    if (height > 4) {
+        height = 4;
+    }
```

Comment:
Why do we limit the height?

Reply:
According to maximum default height for a context image

Reply:
Maybe we should use `config.DEFAULT_CONTEXT_IMAGE_HEIGHT`

Reply:
Maybe..